NitroxServer/GameLogic/Entities/Spawning/BatchEntitySpawner.cs

```diff
@@ -236,6 +238,28 @@ private IEnumerable<Entity> CreateEntityWithChildren(EntitySpawnPoint entitySpaw
             }
         }
 
+        private List<Entity> SpawnChildren(List<EntitySpawnPoint> children, DeterministicBatchGenerator deterministicBatchGenerator)
```

Comment:
It looks like this code is a copy of the above.  Can we make the above code call this?

Reply:
Refactored